### ISSUE TITLE
air quality doesn't has a correct level description

### ISSUE THREAD

**Issue by Greatwallcorner** [breezy-weather/breezy-weather#346]
### Steps to reproduce

1. weather data provider choose china
2. choose a city in china
3. you will see
![图片](https://github.com/breezy-weather/breezy-weather/assets/49832311/83b4a3f7-7098-4842-a041-1c2cf1f3d00f)
4. it seems show right text in english
![图片](https://github.com/breezy-weather/breezy-weather/assets/49832311/d06eea2a-6010-42bf-91db-c1123f1ddc76)

### Expected behavior

air quality 108 should be 轻度污染？
![图片](https://github.com/breezy-weather/breezy-weather/assets/49832311/20a75168-51b4-4ca6-994b-55a1c1020505)
below is refer:
english :https://aqicn.org/scale/
chinese: https://aqicn.org/scale/hk/

### Actual behavior

air quality 108, indicates 中度污染 

### Weather source used

china

### Breezy Weather version

4.4.1-beta

### Android version

13

### Device

samsung s21

### Other details

here is another thing, why doesn't the daily forecast have air quality? I saw it in Geometric Weather.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I read the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document and followed instructions if relevant.
- [X] I have updated the app to **[latest version](https://github.com/breezy-weather/breezy-weather/releases/latest)**.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance)

**Comment by papjul** [comment]
Please read https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md#current-air-quality

**Comment by Greatwallcorner** [comment]
> Please read https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md#current-air-quality
> 
> Tl;dr: we don't use China AQI, we use an universal one.
> 
> If the issue is just labels, then please update to latest version as translations were updated (it's a requirement to use latest version before you open an issue).
> 
> And China source doesn't have daily details but you can update to latest version of Breezy Weather and you will be able to use a secondary weather source for air quality.

i install it from f-droid, last update is 2023-07-30

**Comment by papjul** [comment]
**Acknowledgements**
![image](https://github.com/breezy-weather/breezy-weather/assets/1449159/dfcab965-e2a5-4d04-bb9d-0a0950b9d8e1)

**Release page**
![image](https://github.com/breezy-weather/breezy-weather/assets/1449159/fd561865-bd83-494a-adf5-afd69aa34711)

**Official download links**
![image](https://github.com/breezy-weather/breezy-weather/assets/1449159/60934a96-28af-42e9-b805-579b24ea314c)

**Comment by Greatwallcorner** [comment]
> 

i'm sorry for this, i mean maybe you can synchronous update on fdroid, just want you know

**Comment by papjul** [comment]
App is not on F-Droid official repo, it’s on IzzyOnDroid repo but latest version there is v4.5.3, not v4.4.1.
